tests/unit/test_logging.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import os
+import time
+
+from pip._internal.utils.logging import IndentingFormatter
+
+
+class Test_IndentingFormatter(object):
+    """
+    Test `pip._internal.utils.logging.IndentingFormatter`
+    """
+
+    def setup(self):
+        self.old_tz = getattr(os.environ, 'TZ', None)
+        os.environ['TZ'] = 'UTC'
+        if 'tzset' in dir(time):
+            time.tzset()
+
+    def teardown(self):
+        if self.old_tz:
+            os.environ['TZ'] = self.old_tz
+        else:
+            del os.environ['TZ']
+        if 'tzset' in dir(time):
+            time.tzset()
+
+    def test_formatter_with_timestamp(self, tmpdir):
```

Comment:
I would name this `test_format_with_timestamp()` or `test_format__with_timestamp` since `format()` is the name of the method being tested (formatter is already included in the name of the test class).